examples/07-multiple-files-plugin/soundingGraph.mjs

```diff
@@ -1,8 +1,8 @@
 import overlays from '@windy/overlays';
 import rs from '@windy/rootScope';
 import store from '@windy/store';
-import $ from '@windy/$';
 import _ from '@windy/utils';
+const {$} = _;
```

Comment:
Please, use prettier to keep formatting consistent. Also, imports first, declarations below.